session_2/student_registry/db/migrate/20210114162053_create_students.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :roll_number
+      t.string :branch
+      t.decimal :CGPA, precision: 8, scale: 2
```

Comment:
Should be `:cgpa` as column names are `snake_cased` by Rails conventions.

https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e

Reply:
will change that